inc/notificationtargetticket.class.php

```diff
@@ -119,7 +119,9 @@ function getSubjectPrefix($event = '') {
    **/
    function getContentHeader() {
 
-      if (MailCollector::getNumberOfActiveMailCollectors()) {
+      if (MailCollector::getNumberOfActiveMailCollectors()
```

Comment:
For better performances, I would have check mode before presence of mail collectors.

Reply:
Yep, done :)